acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/publickeys/FilecoinKeyIdentifiersAcceptanceTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.tests.publickeys;
```

Comment:
how do we want to split our ATs? Since each mode is basically a seperate application thinking we should split by eth1, eth2 etc. Instead of functionality. Technically Filecoin doesn't return public keys just fc addresses so bit weird it being in this package.